packages/common/src/ui/components/buttons/standard/ButtonWithIcon.tsx

```diff
@@ -42,19 +42,21 @@ export const ButtonWithIcon: React.FC<ButtonWithIconProps> = ({
 
   return (
     <Tooltip disableHoverListener={!shrink} title={t(labelKey, labelProps)}>
-      <ShrinkableBaseButton
-        disabled={disabled}
-        shrink={shrink}
-        onClick={onClick}
-        variant={variant}
-        color={color}
-        size="small"
-        startIcon={startIcon}
-        {...buttonProps}
-      >
-        {centeredIcon}
-        {text}
-      </ShrinkableBaseButton>
+      <span>
```

Comment:
the tooltip throws a warning if it is around a disabled button because events aren't raised on disabled elements. wrapping the button in a span, which isn't disabled, removes a warning 🤷